envoy/upstream/load_balancer.h

```diff
@@ -271,15 +276,7 @@ using LoadBalancerConfigPtr = std::unique_ptr<LoadBalancerConfig>;
 /**
  * Context information passed to a load balancer factory to use when creating a load balancer.
  */
-class LoadBalancerFactoryContext {
-public:
-  virtual ~LoadBalancerFactoryContext() = default;
-
-  /**
-   * @return Event::Dispatcher& the main thread dispatcher.
-   */
-  virtual Event::Dispatcher& mainThreadDispatcher() PURE;
-};
+using LoadBalancerFactoryContext = Server::Configuration::ServerFactoryContext;
```

Comment:
WDYT of keeping this using option for downstream use cases, but TODO a follow-up clean up PR in Envoy to make it clear that the classes are the same in the long run?

Reply:
Them, I think maybe we can remove this `using` directly because the signature of `loadConfig()` is changed anyway this time.